www/src/app/views/InboxContainer.js

```diff
@@ -1,18 +1,16 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../actions';
 
 import Inbox from '../components/inbox/Inbox';
 import { API_URL } from '../../config';
-import { makeUpdate, get, post } from '../utils';
+import { makeUpdate, post } from '../utils';
 
-export default class InboxContainer extends Component {
+class InboxContainer extends Component {
   constructor(props) {
     super(props);
 
-    const { author } = this.props.params;
-
     this.state = {
```

Comment:
Should we continue to use `this.state` and `this.setState` next to using Redux?  Do you mean this as a temporary measure to split the migration into smaller chunks?